test/testdata/infer/generics/disabled/variance_methods.rb

```diff
@@ -0,0 +1,45 @@
+# typed: true
+
+class A
+  extend T::Sig
+  extend T::Helpers
+  extend T::Generic
+
+  abstract!
+
+  Key = type_member(:in) # error: Classes can only have invariant type members
```

Comment:
Maybe call type params `Co` and `Contra` to make it easier to read?

Reply:
Done, much easier to read!